Dockerfile

```diff
@@ -17,5 +17,6 @@ RUN python3 -m pip install pip poetry --upgrade \
     && printf '#!/bin/bash \n poetry run firewatch $@' > /usr/bin/firewatch \
     && chmod +x /usr/bin/firewatch
 
+COPY --from=zricethezav/gitleaks:v7.6.1 /usr/bin/gitleaks /usr/bin/gitleaks
```

Comment:
latest version is `[v8.18.1](https://github.com/orgs/gitleaks/packages/container/gitleaks/148971421?tag=v8.18.1)`

any reason to use old one?

also I think it's better to use `ghcr.io/gitleaks/gitleaks` , as far as I know there is no rate limit like in dockerhub

Reply:
> latest version is `[v8.18.1](https://github.com/orgs/gitleaks/packages/container/gitleaks/148971421?tag=v8.18.1)`
> 
> any reason to use old one?

Quoting my earlier reply for this question:
> Yes, v7.6.1 is currently the only version [supported by the patterns distribution server](https://source.redhat.com/departments/it/it-information-security/wiki/pattern_distribution_server#supported-gitleaks-versions). I believe newer versions use a different config file format.

> also I think it's better to use `ghcr.io/gitleaks/gitleaks` , as far as I know there is no rate limit like in dockerhub

I did not know that, so I'll test this change. Thank you for the suggestion!

Reply:
Update: I'm not seeing an image tag for the v7.6.1 version on ghcr.io.
Should we build it or leave it as-is?